docker/Dockerfile

```diff
@@ -104,7 +104,7 @@ ENV CONTAINER_SOURCE=/opt/app-root/src
 
 WORKDIR $CONTAINER_SOURCE/
 # Upstream only
-COPY --from=cleanup --chown=1001:1001 $CONTAINER_SOURCE/ ./
```

Comment:
Since the user gets set later on to 1001 I'm fairly certain this is needed.  Any specific reason why the chown was removed?